js/header.js

```diff
@@ -1,20 +1,25 @@
-var navbar = document.getElementsByTagName('nav')[0];
+var navbar = document.getElementById('top-navbar');
 var body = document.getElementsByTagName('body')[0];
 var navButton = document.getElementById('nav-list-button');
 var bannerButton = document.getElementById('banner-button');
 var bannerInput = document.getElementById('banner-email');
 
+
+// navbar gradient
 window.addEventListener('scroll', function(e) {
     if (this.scrollY > 0 || this.innerWidth < 1050) {
         navbar.classList.add('floating');
     } else {
         navbar.classList.remove('floating');
     }
 });
-
-if (window.innerWidth < 1050) {
-    navbar.classList.add('floating');
-}
+window.addEventListener("resize", ()=>{
+    if (window.innerWidth < 1050) {
+        navbar.classList.add('floating');
+    }else if(this.scrollY <= 0){
```

Comment:
You should consider an auto beautifier of some sort for your IDE to clean up spacing like this